services-transport-rsocket/src/main/java/io/scalecube/services/transport/rsocket/RSocketServiceTransport.java

```diff
@@ -49,6 +51,17 @@
     LOGGER.debug("Epoll support: " + preferEpoll);
   }
 
+  private final BiFunction<Channel, Iterator<EventExecutor>, EventExecutor> eventLoopChooserFn =
+      (channel, iterator) -> {
+        while (iterator.hasNext()) {
+          EventLoop eventLoop = (EventLoop) iterator.next();
+          if (eventLoop.inEventLoop()) {
+            return eventLoop;
+          }
+        }
+        return null;
+      };
+
```

Comment:
Instead of making field just put it as `private` method : (call it for example  _chooseEventLoop_ ) 
```java
EventLoop  chooseEventLoop(Channel, Iterator<EventExecutor>) {
  ...
}
```
This way you could conform to java 8  style and  reference this bi function as `this::chooseEventLoop` 
at line [line 89](https://github.com/scalecube/scalecube-services/pull/452/files#diff-4dcf5c95e43f9bdacea335385bc119d9R89)